classes/Colour.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+class Colour
+{
+    private string $colour_name;
+public function getColour(): string
```

Comment:
I think this function needs to be indented

Reply:
Done